tools/lang_demo.py

```
@@ -0,0 +1,388 @@
+    print(f"  {len(L.langs)} languages available, touring {len(langs)}")
+
+    renderer = LangBoard(json.load(open(args.kle, encoding='utf-8')),
+                         unit=args.unit, glyphs=None, bezel=not args.no_bezel,
+                         margin=args.margin, exclude=exclude, dither=False)
+    renderer.compact_halves(lambda mp: 'L' if int(mp.split(',')[0]) < 5 else 'R', gap_px=args.gap)
+
+    # Caption bar under the board.
+    try:
+        cap_font = ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf", 26)
+        sub_font = ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf", 18)
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)